about.html

```diff
@@ -0,0 +1,165 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Playfair+Display&family=Poppins:wght@100;400&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="style.css" />
+  <!-- <link rel="stylesheet" href="debug.css" /> -->
+  <title>About</title>
+</head>
+
+<body>
+  <div class="globe" id="globe">
+    <header>
+      <!-- Top Menu Mobile-->
+      <nav>
+        <div class="top-bar-icon">
+          <a href="#"><img id="hamburger" class="openMenu" src="assets/images/icon-hamburger.svg"
+              alt="Menu Mobile" /></a>
+        </div>
+        <div class="menu mobile-only hidden" id="mobileMenu">
+          <div class="close_btn">
+            <img src="assets/images/icon-cancel.svg" alt="menu cancel" />
+            <div class="top-bar">
+              <div class="top-bar-left">
+                <ul class="mobile-only">
+                  <li><a href="index.html">Language</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#">Events</a></li>
+                  <li><a href="#">Speakers</a></li>
+                  <li><a href="#">Contact</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </nav>
+      <nav class="desktop-only mobile-none">
+        <div class="top-bar">
+          <ul class="sm-menu">
+            <li class="small-menu"><img src="assets/images/facebook.png" alt="logo facebook"></li>
+            <li class="small-menu"><img src="assets/images/twitter.png" alt="logo twitter"></li>
+            <li class="small-menu">English</li>
+            <li class="small-menu">My Page</li>
+            <li class="small-menu">Logout</li>
+          </ul>
+        </div>
+        <div class="bottom-bar">
+          <img src="assets/images/advanced-energy-vector-logo.png" alt="logo" />
+          <ul class="big-menu">
+            <!-- </ul> -->
+            <!-- <ul class="big-menu-right"> -->
+              <li class="big-menu-right">
+                <a href="index.html">Index</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="about.html">About</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#program">Program</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#speak">Speakers</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#sponsors">Sponsors</a>
+              </li>
+              <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <!-- Top Menu Desktop-->
+    </header>
+  </div>
+  <section>
+    <br>
+    <h1 class="event-name">Energy Conference<br>
+      Goiânia, Brazil</h1>
```

Comment:
- [ ] __[OPTIONAL]__ The font size is bigger than what's in the design. Also, this section should be centred, not left aligned.

Yours | Original
---- | ----
![image](https://user-images.githubusercontent.com/16058470/143622466-42e57b96-7065-43d9-8c10-1650e93482e8.png) | ![image](https://user-images.githubusercontent.com/16058470/143622537-a6640173-50d9-41e1-9695-e426c70c74dc.png)